README.md

```diff
@@ -6,3 +6,23 @@ Before build the project install next tools.
 
 1. [Homebrew](https://brew.sh/index_ru.html)
 2. [Sourcery](https://github.com/krzysztofzablocki/Sourcery): ```brew install sourcery```
+
+# Snapshot Tests
+
+## Setup
+Install required frameworks using command in terminal:
+
+`carthage update --platform iOS`
+
+## Launch Commands
+To launch snapshot tests on multiple devices:
+
+`fastlane verify`
+
+To record new reference images:
+
+`fastlane record`
```

Comment:
It is recommended to use fastlane through bundler - could we set it here?

Reply:
yes, sure.